lib/deliver/runner.rb

```diff
@@ -6,7 +6,7 @@ def initialize(options)
       self.options = options
       login
       Deliver::DetectValues.new.run!(self.options)
-      FastlaneCore::PrintTable.print_values(config: options, hide_keys: [:app], title: "deliver #{Deliver::VERSION} Summary")
+      FastlaneCore::PrintTable.print_values(config: options, hide_keys: [:app, 'app_review_information.demo_password'], title: "deliver #{Deliver::VERSION} Summary")
```

Comment:
Why is this not a symbol? I assume because it's generated based on the hash? If so, I think we should change it to be symbols only